pkg/apis/infrastructure/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -16,20 +16,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by main. DO NOT EDIT.
+// autogenerated by controller-gen object, do not modify manually; DO NOT EDIT
```

Comment:
How were these generated with controller-gen? I don't see any changes in the Makefile / bazel

Reply:
controller-gen uses the PROJECT file to detect which version of kubebuilder to use. I updated that to version 2 so no changes to the build need to happen.

Reply:
That makes sense, but how was this file generated? Using which command?

Reply:
oh, I see what you're saying, yes I'll update the makefile, it's using the normal kubebuilder v2 controller-gen command with a modified path